src/ValueFormatters/BasicQuantityUnitFormatter.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace ValueFormatters;
+
+/**
+ * Basic (dummy) unit formatter.
```

Comment:
Why "dummy"? It's not that this implementation doesn't do anything. It just misses localization.